src/styles/components/widgets/_button.scss

```diff
@@ -24,6 +24,7 @@ button {
 }
 
 .button {
+  $c: &;
   &__wrapper {
```

Comment:
wow, thanks, what is this $c: ...

Reply:
ah yeah I was going to use that

with some of my other projects that use scss i tend to write my styles where the name is only declared once, and then I attach it to a variable and can use it later like so:

![carbon 32](https://user-images.githubusercontent.com/11803153/35883752-fcbec302-0b4d-11e8-9694-7a0b90eadab7.png)